test/test_pooling.py

```diff
@@ -411,6 +411,41 @@ def find_one():
         # maxConnecting = unbounded: 30+ connections in ~0.140+ seconds
         print(len(pool.conns))
 
+    def test_csot_timeout_message(self):
+        with self.assertRaises(Exception) as error:
+            with timeout(0.001):
```

Comment:
We'll need to increase this to 500ms to make sure there's enough time to actually run the command (instead of timing out in server selection for example).